touchforms/backend/xformplayer.py

```diff
@@ -202,6 +212,21 @@ def session_state(self):
         state = dict((k, v) for k, v in state.iteritems() if v is not None)
         return state
 
+    def get_xmlns(self):
+        metadata = self.fem.getForm().getMainInstance().getMetaData()
+        return metadata.get("XMLNS")
+
+    def evaluate_xpath(self, xpath):
```

Comment:
not sure i understand how to set up a proper evaluation context fully, but it would be awesome to abstract out touchform state from evaluating xpath. right now we need to have a touchform session id in order to evaluate the xpath, basically rendering this useless to any other parts of HQ. what if this function took a FormEntryModel and an xpath. then in the future i could see a small wrapper around it that would allow HQ to evaluate xpath against an arbitrary form as long as they passed in the proper args to create a FormEntryModel (aka, the xform)